virtcontainers/qemu.go

```diff
@@ -1152,6 +1203,38 @@ func (q *qemu) hotplugBlockDevice(drive *config.BlockDrive, op operation) error
 	return err
 }
 
+func (q *qemu) hotplugVhostUserDevice(vAttr *config.VhostUserDeviceAttrs, op operation) error {
+	err := q.qmpSetup()
+	if err != nil {
+		return err
+	}
+
+	devID := "virtio-" + vAttr.DevID
+
+	if op == addDevice {
+		switch vAttr.Type {
+		case config.VhostUserBlk:
+			err = q.hotplugAddVhostUserBlkDevice(vAttr, op, devID)
```

Comment:
I wonder if it would be safer to change this to:

```go
return q.hotplugAddVhostUserBlkDevice(vAttr, op, devID)
```

The last line of the function can then be simply:

```go
return nil
```

Reply:
Ack. Applied